### PR TITLE
fix(commands): update command handling and ellipses menus for q, toolkit explorer

### PR DESCRIPTION
**Pull request by hayemaxi**
- Hide unnecessary auth commands in amazon q (we should evaluate and remove them in the future)
- Add Q versions of generic toolkit commands for: feedback, going to github, about extension, submit issue, etc.
  - This means the status bar menu github button works now.
- Ellipses for Q menu now independent of the toolkit ext.
- Toolkit explorer buttons no longer throw missing command errors.
- Cleanup package.json to remove unneeded or unused context values.

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amazonq/package.json [hunk 1]
```diff
@@ -454,13 +441,13 @@
                 "command": "aws.amazonq.login",
                 "title": "%AWS.command.login%",
                 "category": "%AWS.title%",
-                "enablement": "isCloud9 || !aws.isWebExtHost",
```
Comment (leigaol) [inline packages/amazonq/package.json:457]: Can you double check whether Q standalone should be supported in C9 and web mode? If yes, then this PR needs to be tested against C9 and web

File: packages/core/src/feedback/vue/submitFeedback.ts [hunk 2]
```diff
@@ -69,16 +69,33 @@ export class FeedbackWebview extends VueWebview {
 
 type FeedbackId = 'AWS Toolkit' | 'CodeWhisperer' | 'Amazon Q' | 'Application Composer'
```
Comment (leigaol) [inline packages/core/src/feedback/vue/submitFeedback.ts:70]: Is 'CodeWhisperer' | 'Amazon Q'  supposed to be merged?

File: packages/core/package.json [hunk 3]
```diff
@@ -1245,13 +1246,13 @@
                     "group": "3_lambda@3"
                 },
                 {
-                    "submenu": "aws.submenu.feedback",
-                    "when": "view =~ /^aws\\./",
+                    "submenu": "aws.toolkit.submenu.feedback",
+                    "when": "view =~ /^aws\\./ && view != aws.AmazonQChatView && view != aws.amazonq.AmazonCommonAuth",
```
Comment (leigaol) [inline packages/core/package.json:1250]: Is it necessary to add `view != aws.AmazonQChatView && view != aws.amazonq.AmazonCommonAuth`? This command belongs to Toolkit, it should not depend on Q states.

Comment (hayemaxi) [reply packages/core/package.json:1250]: Yes actually. Any extension can add views, submenus, options, etc to any other extension- which seems silly at first but makes sense in certain scenarios. If we don't add this, then toolkit will put a duplicate Feedback... submenu item in the ellipses menu for Q.

File: packages/core/src/feedback/vue/submitFeedback.ts (hunk 2 above)
Comment (hayemaxi) [reply packages/core/src/feedback/vue/submitFeedback.ts:70]: No, out of scope for now.

File: packages/amazonq/package.json (hunk 1 above)
Comment (hayemaxi) [reply packages/amazonq/package.json:457]: Q does not need to be supported in C9. Webmode is uncertain, but out of scope for the immediate future. We will revisit these commands.

**Review by leigaol — APPROVED** [review]

**Pull request merged**